rules-tests/CodeQuality/Rector/Identical/FlipTypeControlToUseExclusiveTypeRector/Fixture/fqcn_in_namespace.php.inc

```diff
@@ -8,14 +8,13 @@ class FqcnInNamespace
 {
     public function run()
     {
-        /** @var stdClass|null $stdClass */
```

Comment:
turned this positive test into a native return type... hopefully the test still does what it is expected todo